### ISSUE TITLE
Fabric: /grid command with invalid syntax crashes the game

### ISSUE THREAD

**Issue by liquidev** [gbl/Grid#22]
As in the title.

This applies to any syntax error, such as:
- `/grid` – no arguments. suggestion engine reports this correctly, but hitting enter causes a crash
- `/grid hid` – misspelt `hide`. again, `hide` is suggested but hitting enter by mistake causes a crash
- etc.

Here's the crash log:
```
Reported exception thrown!
net.minecraft.class_148: keyPressed event handler
	at net.minecraft.class_437.method_25412(class_437.java:440)
	at net.minecraft.class_309.method_1466(class_309.java:343)
	at net.minecraft.class_309.method_22678(class_309.java:433)
	at net.minecraft.class_1255.execute(class_1255.java:94)
	at net.minecraft.class_309.method_22676(class_309.java:433)
	at org.lwjgl.glfw.GLFWKeyCallbackI.callback(GLFWKeyCallbackI.java:37)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3101)
	at com.mojang.blaze3d.systems.RenderSystem.flipFrame(RenderSystem.java:102)
	at net.minecraft.class_1041.method_15998(class_1041.java:308)
	at net.minecraft.class_310.method_1523(class_310.java:1068)
	at net.minecraft.class_310.method_1514(class_310.java:681)
	at net.minecraft.client.main.Main.main(Main.java:215)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:224)
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:141)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:27)
Caused by: java.lang.NoSuchMethodError: 'net.minecraft.class_2561 net.minecraft.class_2585.method_10854(net.minecraft.class_124)'
	at net.minecraft.class_746.handler$bae000$onChatMessage(class_746.java:2765)
	at net.minecraft.class_746.method_3142(class_746.java)
	at net.minecraft.class_437.method_25425(class_437.java:313)
	at net.minecraft.class_437.method_25427(class_437.java:306)
	at net.minecraft.class_408.method_25404(class_408.java:98)
	at net.minecraft.class_309.method_1454(class_309.java:345)
	at net.minecraft.class_437.method_25412(class_437.java:435)
	... 19 more
17:35:55.081

---- Minecraft Crash Report ----
// Who set us up the TNT?

Time: 12/25/20, 5:35 PM
Description: keyPressed event handler

java.lang.NoSuchMethodError: 'net.minecraft.class_2561 net.minecraft.class_2585.method_10854(net.minecraft.class_124)'
	at net.minecraft.class_746.handler$bae000$onChatMessage(class_746.java:2765)
	at net.minecraft.class_746.method_3142(class_746.java)
	at net.minecraft.class_437.method_25425(class_437.java:313)
	at net.minecraft.class_437.method_25427(class_437.java:306)
	at net.minecraft.class_408.method_25404(class_408.java:98)
	at net.minecraft.class_309.method_1454(class_309.java:345)
	at net.minecraft.class_437.method_25412(class_437.java:435)
	at net.minecraft.class_309.method_1466(class_309.java:343)
	at net.minecraft.class_309.method_22678(class_309.java:433)
	at net.minecraft.class_1255.execute(class_1255.java:94)
	at net.minecraft.class_309.method_22676(class_309.java:433)
	at org.lwjgl.glfw.GLFWKeyCallbackI.callback(GLFWKeyCallbackI.java:37)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3101)
	at com.mojang.blaze3d.systems.RenderSystem.flipFrame(RenderSystem.java:102)
	at net.minecraft.class_1041.method_15998(class_1041.java:308)
	at net.minecraft.class_310.method_1523(class_310.java:1068)
	at net.minecraft.class_310.method_1514(class_310.java:681)
	at net.minecraft.client.main.Main.main(Main.java:215)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:224)
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:141)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:27)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.class_746.handler$bae000$onChatMessage(class_746.java:2765)
	at net.minecraft.class_746.method_3142(class_746.java)
	at net.minecraft.class_437.method_25425(class_437.java:313)
	at net.minecraft.class_437.method_25427(class_437.java:306)
	at net.minecraft.class_408.method_25404(class_408.java:98)
	at net.minecraft.class_309.method_1454(class_309.java:345)

-- Affected screen --
Details:
	Screen name: net.minecraft.class_408
Stacktrace:
	at net.minecraft.class_437.method_25412(class_437.java:435)
	at net.minecraft.class_309.method_1466(class_309.java:343)
	at net.minecraft.class_309.method_22678(class_309.java:433)
	at net.minecraft.class_1255.execute(class_1255.java:94)
	at net.minecraft.class_309.method_22676(class_309.java:433)
	at org.lwjgl.glfw.GLFWKeyCallbackI.callback(GLFWKeyCallbackI.java:37)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3101)
	at com.mojang.blaze3d.systems.RenderSystem.flipFrame(RenderSystem.java:102)

-- Affected level --
Details:
	All players: 1 total; [class_746['liquide_x'/231747, l='ClientLevel', x=-734.94, y=165.00, z=8.21]]
	Chunk stats: SodiumChunkCache: 80
	Level dimension: minecraft:overworld
	Level spawn location: World: (-737,175,15), Chunk: (at 15,10,15 in -47,0; contains blocks -752,0,0 to -737,255,15), Region: (-2,0; contains chunks -64,0 to -33,31, blocks -1024,0,0 to -513,255,511)
	Level time: 4541151 game time, 4591017 day time
	Server brand: Paper
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.class_638.method_8538(class_638.java:449)
	at net.minecraft.class_310.method_1587(class_310.java:2239)
	at net.minecraft.class_310.method_1514(class_310.java:699)
	at net.minecraft.client.main.Main.main(Main.java:215)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:224)
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:141)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:27)

-- System Details --
Details:
	Minecraft Version: 1.16.4
	Minecraft Version ID: 1.16.4
	Operating System: Linux (amd64) version 5.9.14-arch1-1
	Java Version: 14.0.2, N/A
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 655261024 bytes (624 MB) / 1342177280 bytes (1280 MB) up to 2147483648 bytes (2048 MB)
	CPUs: 12
	JVM Flags: 8 total; -Xss1M -Xmx2G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Fabric Mods: 
		amecsapi: Amecs API 1.1.1+mc1.16-rc1
		architectury: Architectury 1.2.60
		artifice: Artifice 0.13.1+1.16.2-rc1
		autoconfig1u: Auto Config v1 Updated 3.3.1
		cloth-basic-math: Cloth Basic Math 0.5.1
		cloth-client-events-v0: Cloth Client Events v0 1.4.9
		cloth-config2: Cloth Config v4 4.8.3
		cotton-client-commands: Cotton Client Commands 0.4.2+1.14.3-SNAPSHOT
		fabric: Fabric API 0.28.3+1.16
		fabric-api-base: Fabric API Base 0.2.0+ab87788d3a
		fabric-biome-api-v1: Fabric Biome API (v1) 3.1.0+2e23b97c3a
		fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.4+c6a8ea8995
		fabric-command-api-v1: Fabric Command API (v1) 1.0.9+6a2618f53a
		fabric-commands-v0: Fabric Commands (v0) 0.2.0+6a2618f53a
		fabric-containers-v0: Fabric Containers (v0) 0.1.9+a03e98793a
		fabric-content-registries-v0: Fabric Content Registries (v0) 0.2.0+e77439c73a
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.2+b7f9825d3a
		fabric-dimensions-v1: fabric-dimensions-v1 2.0.1+9a6c75813a
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.1+6a2618f53a
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.0+16acbe5b95
		fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.3+a4467d2a3a
		fabric-item-api-v1: Fabric Item API (v1) 1.2.0+6a2618f53a
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.2.1+6a2618f53a
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.1+730711c63a
		fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.0+3fa9f7c595
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.2.0+74cc3b2095
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.1+6a2618f53a
		fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.2+b764ce9995
		fabric-models-v0: Fabric Models (v0) 0.1.1+6a2618f53a
		fabric-networking-api-v1: Fabric Networking API (v1) 1.0.0+4358fbc63a
		fabric-networking-blockentity-v0: Fabric Networking Block Entity (v0) 0.2.7+a03e98793a
		fabric-networking-v0: Fabric Networking (v0) 0.3.1+2a4333d33a
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.9.2+6a2618f53a
		fabric-object-builders-v0: Fabric Object Builders (v0) 0.7.1+6a2618f53a
		fabric-particles-v1: fabric-particles-v1 0.2.2+fb0d9b0e95
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.7.3+be155ae23a
		fabric-renderer-api-v1: Fabric Renderer API (v1) 0.3.3+6a2618f53a
		fabric-renderer-indigo: Fabric Renderer - Indigo 0.4.3+6a2618f53a
		fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 2.2.0+6a2618f53a
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.1.4+6a2618f53a
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.1.12+6a2618f53a
		fabric-rendering-v0: Fabric Rendering (v0) 1.1.1+6a2618f53a
		fabric-rendering-v1: Fabric Rendering (v1) 1.4.0+6a2618f53a
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.3.2+1944f3e03a
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.0+6a2618f53a
		fabric-structure-api-v1: Fabric Structure API (v1) 1.1.1+f1d8af063a
		fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 1.1.0+e77439c73a
		fabric-textures-v0: Fabric Textures (v0) 1.0.5+6a2618f53a
		fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.2.5+6a2618f53a
		fabricloader: Fabric Loader 0.10.8
		grid: Grid 20w17a-fabric0.6.2-1.5
		lightlevel: Light Level 2.0.0
		lithium: Lithium 0.6.0
		lithium-api: Lithium API 0.6.0
		minecraft: Minecraft 1.16.4
		mm: Manningham Mills 2.1
		modmenu: Mod Menu 1.14.13+build.19
		mousewheelie: Mouse Wheelie 1.6.0+mc1.16.4
		org_codehaus_groovy_groovy: groovy 3.0.3
		org_codehaus_groovy_groovy-jsr223: groovy-jsr223 3.0.3
		phosphor: Phosphor 0.7.0+build.10
		slight-gui-modifications: 'Slight' GUI Modifications 1.6.1
		sodium: Sodium 0.1.0
		tweed: Tweed API 3.0.0-beta.21
		voxelmap: VoxelMap 1.10.14
		voyager: Voyager 1.0.0
	Launched Version: fabric-loader-0.10.8-1.16.4
	Backend library: LWJGL version 3.2.2 build 10
	Backend API: Radeon RX Vega (VEGA10, DRM 3.39.0, 5.9.14-arch1-1, LLVM 11.0.0) GL version 4.6 (Compatibility Profile) Mesa 20.3.1, AMD
	GL Caps: Using framebuffer using OpenGL 3.0
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	Graphics mode: fancy
	Resource Packs: slightguimodifications:cts_textures, vanilla, file/FontFix (incompatible), fabric/artifice, fabric/autoconfig1u, fabric/cloth-config2, fabric/fabric, fabric/fabric-api-base, fabric/fabric-biome-api-v1, fabric/fabric-blockrenderlayer-v1, fabric/fabric-command-api-v1, fabric/fabric-containers-v0, fabric/fabric-content-registries-v0, fabric/fabric-crash-report-info-v1, fabric/fabric-events-interaction-v0, fabric/fabric-events-lifecycle-v0, fabric/fabric-game-rule-api-v1, fabric/fabric-item-api-v1, fabric/fabric-item-groups-v0, fabric/fabric-key-binding-api-v1, fabric/fabric-keybindings-v0, fabric/fabric-lifecycle-events-v1, fabric/fabric-loot-tables-v1, fabric/fabric-mining-levels-v0, fabric/fabric-models-v0, fabric/fabric-networking-api-v1, fabric/fabric-networking-blockentity-v0, fabric/fabric-networking-v0, fabric/fabric-object-builder-api-v1, fabric/fabric-object-builders-v0, fabric/fabric-registry-sync-v0, fabric/fabric-renderer-api-v1, fabric/fabric-renderer-indigo, fabric/fabric-renderer-registries-v1, fabric/fabric-rendering-data-attachment-v1, fabric/fabric-rendering-fluids-v1, fabric/fabric-rendering-v0, fabric/fabric-rendering-v1, fabric/fabric-resource-loader-v0, fabric/fabric-screen-handler-api-v1, fabric/fabric-tag-extensions-v0, fabric/fabric-textures-v0, fabric/fabric-tool-attribute-api-v1, fabric/fabricloader, fabric/lithium, fabric/mm, fabric/phosphor, fabric/slight-gui-modifications, fabric/sodium, fabric/voxelmap (incompatible), programer_art
	Current Language: English (US)
	CPU: 12x AMD Ryzen 5 1600 Six-Core Processor
#@!@# Game crashed! Crash report saved to: #@!@# /home/daknus/.minecraft/crash-reports/crash-2020-12-25_17.35.54-client.txt
Saving all world maps.
AL lib: (EE) alc_cleanup: 1 device not closed
17:35:55.081
Process crashed with exit code 255
```

**Comment by gbl** [comment]
Thanks for reporting this. I'll check into it, but I'm afraid this looks like an incompatibility with the cotton-client-commands library that I'm using, so I might have to wait for them to update until I can fix it.

**Comment by gbl** [comment]
I checked the versions of your mods, some of them are outdated. You're using Minecraft 1.16.4, but the 20w17 version of grid - which is for an old weekly pre-release, not the current version. That version included and worked with cotton-client-commands-0.4.2+1.14.3-SNAPSHOT, but that one is severly outdated now, causing the crash.

Please use the 1.16.4 version of Grid. either from https://www.curseforge.com/minecraft/mc-mods/grid/files/3105633 or from https://modrinth.com/mod/grid/version/KwaYDKSx - those should work.

**Comment by liquidev** [comment]
Ah well that makes sense. I thought it should work with 1.16 because CurseForge listed the version I downloaded under the 1.16 group.
![image](https://user-images.githubusercontent.com/16415678/103152069-9a01da80-4784-11eb-961a-0d6b2ab3da15.png)
For some reason in my mind I thought the first version was for Forge… sorry for wasting your time, and thanks for help!
